persistence-zookeeper/src/main/scala/com/mesosphere/usi/storage/zookeeper/PodRecordRepository.scala

```diff
@@ -0,0 +1,48 @@
+package com.mesosphere.usi.storage.zookeeper
+
+import akka.util.ByteString
+import com.mesosphere.usi.core.models.{PodId, PodRecord}
+import com.mesosphere.usi.repository.{PodRecordRepository => PodRecordRepositoryInterface}
+import com.mesosphere.usi.storage.zookeeper.PersistenceStore.Node
+import org.apache.zookeeper.KeeperException.NoNodeException
+
+import scala.async.Async.{async, await}
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+class PodRecordRepository(val store: PersistenceStore) extends PodRecordRepositoryInterface {
+
+  override def create(record: PodRecord): Future[PodId] = async {
+    val data = record.toString
```

Comment:
We need a proper serialisation here.

Reply:
Yup. I was just not sure which one we want.